apps/frontend/components/DisasterTable/CustomCommuneSort.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable complexity */
```

Comment:
Is there really any benefit from using this rule? Maybe we should consider removing it

Reply:
It gives a hint that maybe we should be doing things a bit differently 😅